scripts/collector/oracle/sql/extracts/users.sql

```diff
@@ -14,7 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 spool &outputdir/opdb__users__&v_tag
+<<<<<<< HEAD
+
+=======
```

Comment:
```suggestion
<<<<<<< HEAD

=======
```
These look like merge conflicts?

Reply:
Argh, must have missed it when merging in VS code.